README.md

```diff
@@ -1,11 +1,48 @@
 # ALBA
+This is a Rust library of _Approximate Lower Bound Arguments_ proposed in the [paper](https://iohk.io/en/research/library/papers/approximate-lower-bound-arguments/), May 2024, Eurocrypt'24 by _Pyrros Chaidos, Aggelos Kiayias, Leonid Reyzin, Anatoliy Zinovyev_.
 
-Implementation of _Approximate Lower Bound Arguments_ from the [paper](https://iohk.io/en/research/library/papers/approximate-lower-bound-arguments/) published by IOG Research:
+### Introduction
+ALBA enables a prover who has a large collection of data to convince a verifier that their set includes at least a minimum number of elements that meet a specific condition, called a predicate, even though the prover only reveals a subset of the data.
+By approximating a lower bound on the prover's knowledge, ALBA makes use of a controlled gap between the size of prover's actual knowledge and the threshold of the verifier checks they know.
```

Comment:
This sentence sounds confusing. Here, it says "ALBA makes use of a controlled gap" and the way it does it is "by approximating a lower bound on the prover's knowledge". I think it doesn't make sense logically.

My mild suggestion for the introduction:
1) stating that alba efficiently proves knowledge of elements
2) saying that there is a requirement: gap between what prover knows and what verifier is convinced
3) saying that this why it is called "approximate lower bound"
4) saying that this gap is reason why the prover algorithm can be very efficient